easypost/services/referral_customer_service.py

```diff
@@ -0,0 +1,178 @@
+from typing import (
+    Any,
+    Dict,
+    List,
+    Optional,
+)
+
+import requests
+
+from easypost.constant import TIMEOUT
+from easypost.easypost_object import convert_to_easypost_object
+from easypost.error import Error
+from easypost.models.user import User
+from easypost.requestor import (
+    RequestMethod,
+    Requestor,
+)
+from easypost.services.base_service import BaseService
+
+
+class ReferralCustomerService(BaseService):
+    def __init__(self, client):
+        self._client = client
+        self._model_class = User.__name__
+
+    def create(self, **params) -> Dict[str, Any]:
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏ Should this function accept an explicit `partner_api_key` parameter?

Reply:
No? You create a client with the partner's API key